docs/deployment/uwsgi_cookiecutter_2__emperor.rst

```diff
@@ -169,6 +178,8 @@ Running Your App via the Emperor
         emperor = /etc/uwsgi/vassals
         limit-as = 1024
         logto = /var/log/uwsgi/emperor.log
+        uid = ubuntu
+        gid = www-data
 
```

Comment:
Added uid & gid here so emperor runs as ubuntu/www-data as well. (Otherwise log file warns that you are running as superuser)